### DISCUSSION TITLE
git-tags datasource is adding unnessesary "==" for the newValue update with utilized with the Elixir Manager

### DISCUSSION THREAD

**Discussion by christopherbox** [renovatebot/renovate#31998] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Docker - renovate:38.21

### Please tell us more about your question or problem

When Renovate runs with the elixir manager and finds any dependencies that utilize git tags for their versions, it will produce a result with an incorrect tag when the tag is set to a static matched value.

In the following example, if there is a service that pins a tag that is a version, then the end result pre-appends a "==" sign to the proposal.

An example of a mix.exs file being updated.

Before Proposal

---
mix.exs file:
```

defmodule NotFullExampleWeb.MixProject do
  use Mix.Project
 ...
    [
      {:auth_header_plug, git: "git@github.com:PrivateOrg/auth_header_plug", tag: "1.7.0"},
      {:stats_owl, git: "git@github.com:PrivateOrg/stats_logger.git", tag: "2.4.0"},

    ]
  end
end
```

After Proposal:

---

This PR contains the following updates:

| Package | Update | Change |
|---|---|---|
| [auth_header_plug](https://togithub.com/PrivateOrg/auth_header_plug) | minor | `1.7.0` -> `== 1.9.1` |
| [stats_logger](https://togithub.com/PrivateOrg/stats-logger) | minor | `2.4.0` -> `== 2.6.0` |

---
End result of the proposal for the mix.exs file:
```

defmodule NotFullExampleWeb.MixProject do
  use Mix.Project
 ...
    [
      {:auth_header_plug, git: "git@github.com:PrivateOrg/auth_header_plug", tag: "==1.9.1"},
      {:stats_owl, git: "git@github.com:PrivateOrg/stats_logger.git", tag: "==2.6.0"},

    ]
  end
end
```

### Logs (if relevant)

<details><summary>Logs</summary>

```
{
  "name": "renovate",
  "hostname": "bb68e3cc0614",
  "pid": 55,
  "level": 20,
  "logContext": "eEWNdUiMgh38_HdnE2UBg",
  "repository": "PrivateOrg/testing-wild-app",
  "branchesInformation": [
    {
      "branchName": "renovate/all-minor-patch",
      "prNo": 1,
      "prTitle": "chore(deps): update all non-major dependencies",
      "result": "done",
      "upgrades": [
        {
          "datasource": "git-tags",
          "depName": "auth_header_plug",
          "displayPending": "",
          "fixedVersion": "1.7.0",
          "currentVersion": "1.7.0",
          "currentValue": "1.7.0",
          "newValue": "== 1.9.1",
          "newVersion": "1.9.1",
          "packageFile": "apps/testing_wild_app_web/mix.exs",
          "updateType": "minor",
          "packageName": "git@github.com:PrivateOrg/auth_header_plug"
        },
        {
          "datasource": "git-tags",
          "depName": "stats_logger",
          "displayPending": "",
          "fixedVersion": "2.4.0",
          "currentVersion": "2.4.0",
          "currentValue": "2.4.0",
          "newValue": "== 2.6.0",
          "newVersion": "2.6.0",
          "packageFile": "apps/testing_wild_app_web/mix.exs",
          "updateType": "minor",
          "packageName": "git@github.com:PrivateOrg/stats-logger.git"
        }
      ]
    }
  ],
  "msg": "branches info extended",
}

```

</details>

**Comment by RahulGautamSingh** [comment]
Please link the discussion/issue it is the duplicate of. It will help other users.
